.gitignore

```diff
@@ -9,3 +9,4 @@ package-lock.json
 /scratchpad
 yarn-error.log
 .cache
+**/yarn.lock
```

Comment:
This will match the root `yarn.lock` too, right? Maybe `*/**/yarn.lock` is what we want?